00_core.ipynb

```diff
@@ -436,11 +436,14 @@
    "outputs": [],
    "source": [
     "#| export\n",
-    "def create_libs_datasets(libs, # List of libraries to create datasets for (ie ['fastcore','fastkaggle']\n",
+    "def create_libs_datasets(libs, # library or list of libraries to create datasets for (ie 'fastcore or ['fastcore','fastkaggle']\n",
     "                         lib_path, # Local path to dl/create dataset\n",
-    "                         username # You username\n",
+    "                         username, # You username\n",
+    "                         clear_after=False # Delete local copies after sync with kaggle?\n",
     "                        ):\n",
     "    '''For each library, create or update a kaggle dataset with the latest version'''\n",
+    "    if type(libs)==str: libs = [libs] \n",
```

Comment:
FYI, IIRC you can just use `L(libs)` and it automatically does this - no need for the conditional